dev/io.openliberty.jakarta.mail.2.0/bnd.bnd

```diff
@@ -0,0 +1,83 @@
+#*******************************************************************************
+# Copyright (c) 2017, 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+bVersion=2.0
+
+
+bSymbolicName=io.openliberty.jakarta.mail.2.0
+
+Bundle-Name: Jakarta Mail 2.0 API
+Bundle-SymbolicName: io.openliberty.jakarta.mail.2.0
+Bundle-Description: Jakarta Mail 2.0 API; This feature allows applications to utilize the Jakarta Mail 2.0 API. 
+
+WS-TraceGroup: Mail
+
+
+## Resources that were moved from the original javax.mail.jar to
+## the new feature jar, included the mailcap file which is needed
+## for the app-resources header
+
+Include-Resource: \
+  @${repo;com.ibm.ws.com.sun.mail.jakarta.mail.2.0}!/META-INF/mailcap, \
+  @${repo;com.ibm.ws.com.sun.mail.jakarta.mail.2.0}!/META-INF/gfprobe-provider.xml, \
+  @${repo;com.ibm.ws.com.sun.mail.jakarta.mail.2.0}!/META-INF/javamail.charset.map, \
+  @${repo;com.ibm.ws.com.sun.mail.jakarta.mail.2.0}!/META-INF/javamail.default.address.map, \
+  @${repo;com.ibm.ws.com.sun.mail.jakarta.mail.2.0}!/META-INF/javamail.default.providers  
+
+Service-Component: \
+  io.openliberty.jakarta.mail_2_0_${replace;${bVersion};\\.;_}.ResourceProvider; \
+    implementation:=com.ibm.wsspi.classloading.ResourceProvider; \
+    provide:=com.ibm.wsspi.classloading.ResourceProvider; \
+    configuration-policy:=ignore; \
+    properties:= "resources=${app-resources}"
+
+-dsannotations=io.openliberty.jakarta.mail.internal.MailSessionService, \
+  io.openliberty.jakarta.mail.internal.injection.MailSessionDefinitionInjectionProcessorProvider, \
+  io.openliberty.jakarta.mail.internal.injection.MailSessionResourceFactoryBuilder
+
+## The only file that is used by the app-resources is the
+## mailcap file, this is because it is used by JAF to assess
+## what type of content the Message classes uses with its DataHandler
+## I originally included the other files, but commented them out for now
+## as I'm not entirely sure they are needed for the feature to work properly
+app-resources= \
+  META-INF/mailcap 
+
+Private-Package: \
+	io.openliberty.jakarta.mail.internal.injection,\
+	io.openliberty.jakarta.mail.internal,\
+	com.ibm.ws.jakarta.mail.resources,\
+	com.ibm.ws.config.xml.internal.nester
+
+Import-Package: \
+  !*.internal.*, \
+  *
+
+Export-Package: io.openliberty.jakarta.mail
```

Comment:
If this is an Open Liberty API that is being exported from this bundle then I think it is confusing to include `jakarta` in the package? I think this would be better as `io.openliberty.mail`??